flexget/plugins/services/myepisodes.py

```diff
@@ -1,27 +1,27 @@
 from __future__ import unicode_literals, division, absolute_import
 from builtins import *  # pylint: disable=unused-import, redefined-builtin
-from future.moves.urllib import request
-from future.moves.urllib import parse
-from future.moves.urllib.error import URLError
 
 import logging
 import re
 from datetime import datetime
-import http.cookiejar
 
 from sqlalchemy import Column, Integer, String, DateTime
 
-from flexget import db_schema, plugin
+from flexget import plugin
 from flexget.event import event
+from flexget.utils import requests
+from flexget.db_schema import versioned_base
+
 
 try:
     from flexget.plugins.internal.api_tvdb import lookup_series
```

Comment:
Should remove this import and use `plugin.get_plugin_by_name('api_tvdb').instance.lookup_series` where applicable instead. It will raise a dependencyerror if it's not found.